File: 프로그래머스/0/120905. n의 배수 고르기/n의 배수 고르기.java
class Solution {
    public int[] solution(int n, int[] numlist) {
        int num = 0;
        for (int i = 0; i < numlist.length; i++) {
            if (numlist[i] % n == 0) {
                num++;
            } 
        }
        
        int[] answer = new int[num];
        int answer_num = 0;
        
        for (int j = 0; j < numlist.length; j++) {
            if (numlist[j] % n == 0) {
                // num++;
                answer[answer_num++] = numlist[j]; // 배열에 넣고나서 후위증가
                // answer_num++;
            } 
        }
        return answer;
    }
}

File: 프로그래머스/0/120905. n의 배수 고르기/n의 배수 고르기.js
function solution(n, numlist) {
    var answer = [];
    
    for (i = 0; i < numlist.length; i++) {
        if (numlist[i] % n != 0) {
            numlist.splice(i, 1);
            i--;
        }
                
        answer = numlist;

    }
    
    return answer;
}